_collections/_news/2024-12-14-2024-25s-semester-1-overview.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: post
+published: true
+title: CSS' Best Semester - 2024/25 Semester 1 Overview
+thumbnail: /assets/images/2024-semester-1/thumbnail.png
+header-image: /assets/images/2024-semester-1/banner.png
+---
+What a blazing start it's been for CSS - there's so much that's happened, even I've struggled to keep up! 
+
+If you need a quick reminder on what's happened, let's go over everything...
+
+# What's been on this semester
+Since our welcome week, we've been running a series of new events mixed in with old ones, both with old friends and new
+ones. We kicked things off with our classic Cream Tea event, joined by TeaSoc! Even if we ended up buying Sainsbury's
+full stock of scones and forgot the cream, it was still an incredibly successful event, giving us the chance to catch
+up with everyone after the chaos of welcome week. Thank you once again to TeaSoc for hosting this event with us <3
```

Comment:
```suggestion
Since our welcome week, we've been running a mix of new and recurring events, kicking off first with Cream Tea with TeaSoc! 
Even if we ended up buying Sainsbury's full stock of scones and forgot the cream, it was still an incredibly successful 
event, giving us the chance to catch up with everyone after the chaos of welcome week. Thank you once again to
TeaSoc for hosting this event with us <3
```